### ISSUE TITLE
iOS support

### ISSUE THREAD

**Issue by ghost** [evilcorpltd/aTox#642]
This topic will still anybody create, so let it be me.
I understand that the main users are android, but sometimes we want to keep a safe dialogue with people who have an iPhone 
Therefore, I believe that this class protocol must have a client for  iOS.

## Step
Create a porting plan
Porting
Test
Add to Apple Store

**Comment by robinlinden** [comment]
I'm already working on an iOS version.

Things that need to be done:
- [ ] Switch from Room to SQLDelight
- [ ] Port domain and core modules to Kotlin multiplatform
- [ ] Add native iOS GUI or Jetpack Compose GUI

**Comment by amirmahdi-nsh** [comment]
Unfortunately we will also have some problems in receiving messages when the app is in background. Because iOS will kill every process which is running in background If it runs for more than 10 minutes :(
Also [Antidote](https://github.com/Antidote-for-Tox/Antidote/blob/develop/FAQ/en.md#why-dont-i-receive-push-notifications-in-background) and [Briar](https://code.briarproject.org/briar/briar/-/wikis/FAQ#will-there-be-an-ios-version-of-briar) have the same problem.
But I think eventually it should be solve because the Bridgefy also have iOS version and solved this problem(but due it's but open we couldn't understand how!?)

**Comment by xbvrhlmb** [comment]
> Unfortunately we will also have some problems in receiving messages when the app is in background. Because iOS will kill every process which is running in background If it runs for more than 10 minutes :(
> Also [Antidote](https://github.com/Antidote-for-Tox/Antidote/blob/develop/FAQ/en.md#why-dont-i-receive-push-notifications-in-background) and [Briar](https://code.briarproject.org/briar/briar/-/wikis/FAQ#will-there-be-an-ios-version-of-briar) have the same problem.
> But I think eventually it should be solve because the Bridgefy also have iOS version and solved this problem(but due it's but open we couldn't understand how!)

Yes, this is an important point, is it possible to refresh the process every 5 minutes? :)

**Comment by amirmahdi-nsh** [comment]
>is it possible to refresh the process every 5 minutes?

I don't think so. But there should be a solution because Bridgefy works

**Comment by zoff99** [comment]
@amirmahdi-nsh actually [Antidote](https://github.com/Zoxcore/Antidote) does not have that issue anymore.
and it's not 10 minutes (that was a long time ago) but now its less than 30 seconds.

**Comment by wyldcrd** [comment]
I meant to post my comment in a different repo i feel like such a dingus right now 😔
